src/main/java/jpos/config/JposEntryRegistryEvent.java

```diff
@@ -28,6 +28,11 @@
 public class JposEntryRegistryEvent extends EventObject
 {
     /**
+	 * 
```

Comment:
Meaningful JavaDoc comment missing

Reply:
I removed the empty comment. It had been added automatically by Eclipse when generating the serial UID to solve a compiler warning.